media/server/main/source/MediaPipelineServerInternal.cpp

```diff
@@ -28,6 +28,7 @@
 
 namespace
 {
+constexpr std::chrono::milliseconds kNeedMediaDataResendTime{100};
```

Comment:
pedantic: can we call it kNeedMediaDataResendTimeMs please

Reply:
done